adobe_vipm/flows/helpers.py

```diff
@@ -164,6 +165,10 @@ def __call__(self, client, context, next_step):
         context.currency = context.order["agreement"]["listing"]["priceList"]["currency"]
         context.customer_data = get_customer_data(context.order)
         context.market_segment = get_market_segment(context.product_id)
+        if is_large_government_agency_type(context.product_id):
+            context.customer_data[Param.AGENCY_TYPE.value] = get_agency_type(context.order)
+            context.customer_data["is_lga_account"] = True
```

Comment:
I'm not sure that we need one more way to detect if it is an lga account or not. Now you have 2 ways todo it:

1. is_large_government_agency_type function
2. customer_data["is_lga_account"]